index.js

```diff
@@ -96,6 +96,15 @@ if (debugMode) {
     console.log("DEBUG_MODE is active. To protect privacy, do not use in production.")
 }
 
+const ipv6_lcl = parseInt(process.env.IPV6_LOCALIZE);
+if (ipv6_lcl) {
+    if (ipv6_lcl < 1 || ipv6_lcl > 8) {
+        console.debug("IPV6_LOCALIZE must be an integer between 1 and 8");
+    } else {
+        console.debug("IPV6 source IPs will be localized to", ipv6_lcl, "segment(s)");
+    }
+}
```

Comment:
If `IPV6_LOCALIZE` does not include numbers `ipv6_lcl` is `NaN`. Also console.debug should not be used in production. Please, replace with:
```javascript
const ipv6_lcl = parseInt(process.env.IPV6_LOCALIZE);
if (ipv6_lcl || isNaN(ipv6_lcl)) {
    if (ipv6_lcl < 1 || ipv6_lcl > 8 || isNaN(ipv6_lcl)) {
        console.error("IPV6_LOCALIZE must be an integer between 1 and 8");
        return;
    }
    console.log("IPv6 client IPs will be localized to", ipv6_lcl, ipv6_lcl > 1 ? "segments" : "segment");
}
```

Reply:
Thanks, I will make some changes and resubmit

Reply:
Please correct me if I'm wrong, but: The `parseInt()` makes any input that can't be converted to an integer return a `NaN`. So there's no need for the additional `|| isNaN(...)`. `if (ipv6_lcl)` evaluates to FALSE if `ipv6_lcl` is NaN anyway, so the block will be correctly skipped.

Reply:
@schlagmichdoch Nevermind my last comment. I see why you want it this way, so that the error message is logged for invalid input. I'll update...